### PR TITLE
created ashishtom.json

### PR DESCRIPTION
**Pull request by alceil**
Fixes #

## Proposed Changes

  -
  -
  -

## Screenshots

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/260"></a>

### CODE REVIEW

File: public/data/ashishtom.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/ashishthomas6",
+        "icon": "twitter"
+      }
+    ]
+  }
+
```
Comment (schmelto) [inline public/data/ashishtom.json:18]:
```suggestion
```

File: public/data/ashishtom.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Ashish Thomas",
+    "bio": "Flutter and web developer",
+    "avatar": "https://github.com/alceil.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "http://github.com/alceil",
```
Comment (schmelto) [inline public/data/ashishtom.json:8]:
```suggestion
        "url": "https://github.com/alceil",
```

Comment (alceil) [reply public/data/ashishtom.json]: done

**Review by aayush89890 — APPROVED** [review]
Looks Good.

File: public/data/ashishtom.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        "url": "https://github.com/alceil",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/ashishthomas6",
+        "icon": "twitter"
+      }
+    ]
+  }
```
Comment (Panquesito7) [inline public/data/ashishtom.json:17]: Missing an endline.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/ashishtom.json (hunk 3 above)
Comment (alceil) [reply public/data/ashishtom.json]: done

File: public/data/ashishtom.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+        "url": "https://github.com/alceil",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/ashishthomas6",
+        "icon": "twitter"
+      }
+    ]
+}
```
Comment (Panquesito7) [inline public/data/ashishtom.json:17]: Still missing an endline.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/ashishtom.json (hunk 4 above)
Comment (alceil) [reply public/data/ashishtom.json]: done

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**